tools/test.py

```diff
@@ -89,33 +88,30 @@ def main(argv):
 
     enable_ansi_colors()
 
-    with spawn():
-        test_cases = [
-            TestSetup,
-            TestUtil,
-            TestTarget,
-            JsUnitTests,
-            FetchTest,
-            FmtTest,
-            TestIntegrations,
-            TestRepl,
-            TestDenoDir,
-            TestBenchmark,
-        ]
-        # These tests are skipped, but to make the test output less noisy
-        # we'll avoid triggering them.
-        if os.name != 'nt':
-            test_cases.append(TestIsTty)
-            test_cases += permission_prompt_tests()
-            test_cases += complex_permissions_tests()
-
-        suite = unittest.TestSuite([
-            unittest.TestLoader().loadTestsFromTestCase(tc)
-            for tc in test_cases
-        ])
-
-        result = ColorTextTestRunner(
-            verbosity=args.verbosity + 1, failfast=args.failfast).run(suite)
+    test_cases = [
+        TestSetup,
+        TestUtil,
+        TestTarget,
+        JsUnitTests,
+        FetchTest,
+        FmtTest,
+        TestIntegrations,
+        TestRepl,
+        TestDenoDir,
+        TestBenchmark,
+        TestIsTty,
+    ]
+    test_cases += permission_prompt_tests()
+    test_cases += complex_permissions_tests()
```

Comment:
Note: This does make the windows tests very noisy!

Reply:
Yep, wanted to see output for myself. I'm really 50/50 on this one, not really a fan of second conditional here and we're running tests with verbose flag anyway so I guess I'm leaning towards leaving it as is